### PR TITLE
[MAINTENANCE] Update aloglia scripts for new fields and replica indices

### PR DESCRIPTION
**Pull request by rpanwar-winfo**
Changes proposed in this pull request:
- Add new fields to upload to algolia exp_type, coverage_score
- Add support for replica indices for sorting
- Changes for reading .env files to load environment variables from file (for local development and testing)

- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: assets/scripts/AlgoliaScripts/upload_s3_expectation_to_algolia.js [hunk 1]
```diff
@@ -15,10 +53,11 @@ loadFromS3(expecS3URL).then(response => {
             return;
         }
         console.log("Formatted expectation sample: ", algDataset[0]);
+        // return;
         deleteIndex(algDataset);
     }
 }).catch((error) => {
-    console.log('Error fetching data from s3', error);
+    console.log('Error ferching data from s3', error);
```
Comment (kenwade4) [inline assets/scripts/AlgoliaScripts/upload_s3_expectation_to_algolia.js]: Typo for fetching

**Review by kenwade4 — CHANGES_REQUESTED** [review]
Looks good. Just a typo fix needed.

File: assets/scripts/AlgoliaScripts/upload_s3_expectation_to_algolia.js (hunk 1 above)
Comment (rpanwar-winfo) [reply assets/scripts/AlgoliaScripts/upload_s3_expectation_to_algolia.js]: Done.

**Review by kenwade4 — APPROVED** [review]

**Comment by kyleaton** [comment]
@cla-bot check

**Review by kenwade4 — APPROVED** [review]

**Pull request merged**